helm-charts/vsts-agent/values.yaml

```diff
@@ -0,0 +1,37 @@
+# Default values for vsts-agent.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+image:
+  repository: airsharedcontainers.azurecr.io/kube-agent
+  # tag has to be a string, please quote it
+  tag: "1094096"
```

Comment:
Can we templatize this tag?

Reply:
yes. This is the default one. Values in this file can be customized during install